MSAL/src/public/MSALSilentTokenParameters.h

```diff
@@ -64,6 +64,13 @@ Initialize a MSALSilentTokenParameters with scopes and account.
 */
 - (instancetype)initWithScopes:(NSArray<NSString *> *)scopes NS_UNAVAILABLE;
 
+/**
+ Key-value pairs to pass to the authentication server during
+ the silent authentication flow. This should not be url-encoded value.
+ */
+@property (nonatomic, nullable) NSDictionary <NSString *, NSString *> *extraQueryParameters;
```

Comment:
If we decide to support EQP for silent flows, we should move it to the parent `MSALTokenParameters` class.